commands/user.go

```diff
@@ -770,19 +772,20 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 }
 
 func verifyUserEmailWithoutTokenCmdF(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	var result *multierror.Error
 	users, err := getUsersFromArgs(c, userArgs)
 	if err != nil {
-		printer.PrintError(err.Error())
+		result = multierror.Append(result, err)
 	}
 
 	for _, user := range users {
 		if newUser, _, err := c.VerifyUserEmailWithoutToken(user.Id); err != nil {
-			printer.PrintError(fmt.Sprintf("unable to verify user %s email: %s", user.Id, err))
+			result = multierror.Append(result, fmt.Errorf("unable to verify user %s email: %s", user.Id, err))
```

Comment:
Please use `%w` to wrap errors in `fmt.Errorf`.

Reply:
should i instead change err to err.Error()?
since tests would be expecting the string of the error?

Reply:
No, just turn the second `%s` into a `%w`. It will behave like `%s` but internally maintains the original error, which might be useful at a later time.

Reply:
done